<launch>
    <arg name="wpt_path" default="$(find Morai_Woowa)/path/test_path.csv" />

    <node name="state_node" pkg="Morai_Woowa" type="state_node" output="screen">
       <param name="/waypoint_file" value="$(arg wpt_path)"/>
    </node>

</launch>
